gauntlet.sh

```diff
@@ -1,15 +1,17 @@
+#!/bin/bash
+
 # Integration test for copy on demand
 # Tests:
 #  * Full read/write
 #  * Read/write blocks offset by one byte to test for off-by-one errors
 #  * Read/write random sizes and offsets $fuzz_count times
 
 block_size=4096 # 4k bytes
-create_blocks=100 # How many $block_size blocks to generate for test
+create_blocks=262144 # How many $block_size blocks to generate for test
 source_file='./test.dat'
 backing_file='./test.dat.bak'
-cod_executable='/usr/bin/copy-on-demand'
-nbd_device='/dev/nbd0'
+cod_executable='/home/behrlich/copyondemand/copy-on-demand'
```

Comment:
This is a personal file path.
Can you either update this to interact with the [binary generated within the repository](https://github.com/datto/copyondemand/blob/main/Makefile#L47) or revert this line (and assume that the install maketarget ran)?

Reply:
👍 yup all WIP issues. I hadn't decided whether there should be a gauntlet-nbd and gauntlet-kmod or have them all merged together. And yeah not sure why this was using the deb path, it should be using the `.bin/copyondemand` path, I'll take care of that.